inventories/local/group_vars/faucet.yml

```diff
@@ -0,0 +1,4 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022-2023, E36 Knots
+---
+avalanche_faucet_config: []
```

Comment:
```suggestion
avalanche_faucet_config: {}
```

Reply:
Applied and renamed to `avalanche_faucet_chains`.